services/src/main/java/org/fao/geonet/api/regions/metadata/MetadataRegionSearchRequest.java

```diff
@@ -196,9 +200,25 @@ private MultiPolygon getMultiPolygon(HashSet<Polygon> polygons) {
     }
 
     Region parseRegion(Id mdId, Element extentObj) throws Exception {
```

Comment:
Should we remove that also which is schema dependent? Here it only works for ISO19139

Reply:
To do that, the extract-gml in all schema plugins would need to be able to handle xml fragments which was soemthing outside the scope of what I was looking at (the spatial extent for the metadata record as a whole).

Reply:
Sure. I was not sure about the usage of parseRegion (maybe we could simply remove it ?) I was also looking to Elasticsearch branch where extract-gml is basically removed (geometries are stored in the index and handled by the main XSLT for indexing).

Reply:
I'm treading carefully here, trying not to break (or remove) anything - only adding.  Not really sure who may or may not be using these apis.  I'm able to work out whether geonetwork core uses something or not, but not what third parties use.  If parseRegion is not used - yes would be good to remove.

Reply:
Sure. We can do clean up in elastic branch.